src/features/at-method.md

```diff
@@ -0,0 +1,43 @@
+---
+title: '`at` method for relative indexing'
+author: 'Shu-yu Guo ([@_shu](https://twitter.com/_shu))'
+avatars:
+  - 'shu-yu-guo'
+date: 2021-07-12
+tags:
+  - ECMAScript
+description: 'JavaScript now has a relative indexing method for Arrays, TypedArrays, and Strings.'
+---
+
+The new `at` method on `Array.prototype`, the various TypedArray prototypes, and `String.prototype` makes accessing an element nearer to the end of the collection easier and more succinct.
+
+Accessing the Nth element from the end of a collection is a common operation. However, the usual ways to do so are verbose, like `my_array[my_array.length - N]`, or might not be performant, like `my_array.slice(-N)[0]`. The new `at` method makes this operation more ergonomic by interpreting negative indices to mean "from the end". The previous examples may be expressed as `my_array.at(-N)`.
```

Comment:
Wanna mention the casual `length - N` bug (that you just did)?

Reply:
Wait, what's the bug?

Reply:
Oh man, I made the big, not you. I thought you had to do `length - N - 1`. 😅

Reply:
Ah yeah, proposal to special case -0? :P